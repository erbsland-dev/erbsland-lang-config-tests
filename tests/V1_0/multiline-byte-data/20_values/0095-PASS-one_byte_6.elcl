# Erbsland Configuration Language Test File
[byte_data_values]
value 17:
    <<<hex
    01
    >>>
