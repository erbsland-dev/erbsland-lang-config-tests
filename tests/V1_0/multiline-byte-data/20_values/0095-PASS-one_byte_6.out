byte_data_values = SectionWithNames()
byte_data_values.value_17 = Bytes(01)
